frontend/src/features/housingCompany/HousingCompanyListPage.tsx

```diff
@@ -69,11 +69,15 @@ const HousingCompanyResultsList = ({filterParams}): JSX.Element => {
                 isLoading={isLoading}
             >
                 <LoadedHousingCompanyResultsList data={data as IHousingCompanyListResponse} />
-                <ListPageNumbers
-                    currentPage={currentPage}
-                    setCurrentPage={setCurrentPage}
-                    pageInfo={data?.page}
-                />
+                {data !== undefined ? (
+                    <ListPageNumbers
+                        currentPage={currentPage}
+                        setCurrentPage={setCurrentPage}
+                        pageInfo={data.page}
+                    />
+                ) : (
+                    <></>
+                )}
```

Comment:
No need to check if data is undefined, QueryStateHandler already does that, you can do this instead:
```ts
<ListPageNumbers
    currentPage={currentPage}
    setCurrentPage={setCurrentPage}
    pageInfo={(data as IHousingCompanyListResponse).page}
/>
```

Reply:
Nice! I have to admit it didn't feel right using a ternary operator simply to please type checking and/or the linter. 😅